unblob/dependencies.py

```diff
@@ -0,0 +1,24 @@
+import shutil
+from typing import List, Type
+
+from .models import Handler
+
+INSTALLED = "✓"
+NOT_INSTALLED = "✗"
+
+
+def is_installed(command: str) -> bool:
+    return bool(shutil.which(command))
```

Comment:
From the documentation: 
> Return the path to an executable which would be run if the given cmd was called. If no cmd would be called, return None.

It would be more precise to use the `None` return value telling the absence, but also, this function is so simple, you don't need a function for it. I would just inline this as a variable. It is only used once.